runner/ollamarunner/runner.go

```diff
@@ -89,7 +89,7 @@ type NewSequenceParams struct {
 	numPredict int
 	stop       []string
 	numKeep    int32
-	sampler    sample.Sampler
+	sampler    *sample.Sampler
```

Comment:
Is there a reason for making this a pointer? Avoiding copy? If so we should have a sanity check so we dont cause a panic on `seq.sampler.Sample(...` just in case something changes and it dies

Reply:
Good catch. I'll remove the pointer.